cmd/update.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/blang/semver"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func selfUpdate() {
+	current := version
+	latest, found, err := selfupdate.DetectLatest("screwdriver-cd/sd-local")
+	if err != nil {
+		fmt.Println("Error occurred while detecting version:", err)
+		return
+	}
+	fmt.Println("Current version:", current)
+	if current == "dev" {
+		return
+	}
+	v := semver.MustParse(current)
+	if !found || latest.Version.LTE(v) {
+		fmt.Println("Current version is the latest")
+		return
+	}
+	fmt.Print("Do you want to update to ", latest.Version, "? (y/n): ")
+	input, err := bufio.NewReader(os.Stdin).ReadString('\n')
+	if err != nil || (input != "y\n" && input != "n\n") {
```

Comment:
I suggest to create default choices.
If created default choices and default is "Yes" then user input is only for enter key.

Reply:
Maybe you can try create a "-y" option that does not show the question and always selects "Yes".
This can be useful if the user is running from Script.
We may not have such a user, but...

Reply:
Thank you review.
I create `-y --yes ` flags.